code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -51,6 +51,11 @@
 		break
 	if(rune)
 		limit = MAX_BLOODCHARGE
+	var/empowering_channel_time = 10 SECONDS - (rune ? 6 SECONDS : 0 SECONDS)
+	var/list/signal_return_list = list("limit_data" = limit, "rune_data" = rune, "speed_data" = empowering_channel_time)
+	SEND_SIGNAL(owner, COMSIG_LIVING_CULT_EMPOWER, signal_return_list)
+	limit = signal_return_list["limit_data"]
+	empowering_channel_time = signal_return_list["speed_data"]
```

Comment:
Same here, "limit_data" could like, be a var on the cult datum? and "speed_data" be a multiplier var on /datum/action/innate/cult/blood_magic`?